### PR TITLE
Adds admin button to change central command's name.

### PR DESCRIPTION
**Pull request by Shadowlight213**
The proc do to it already existed. Might as well add a button for it.

### CODE REVIEW

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -482,6 +482,18 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	message_admins("[key_name_admin(src)] has created a command report")
 	feedback_add_details("admin_verb","CCR") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_change_command_name()
+	set category = "Special Verbs"
+	set name = "Change Command Name"
+	if(!holder)
+		src << "Only administrators may use this command."
+		return
+	var/input = input(usr, "Please a new name for Central Command.", "What?", "") as text|null
```
Comment (Jordie0608) [inline code/modules/admin/verbs/randomverbs.dm]: 'Please a new'

File: code/modules/admin/verbs/randomverbs.dm [hunk 2]
```diff
@@ -482,6 +482,18 @@ Traitors and the like can also be revived with the previous role mostly intact.
...
+	set category = "Special Verbs"
+	set name = "Change Command Name"
+	if(!holder)
+		src << "Only administrators may use this command."
+		return
+	var/input = input(usr, "Please a new name for Central Command.", "What?", "") as text|null
+	if(!input)
+		return
+	change_command_name(input)
+	message_admins("[key_name_admin(src)] has created a changed Central Command's name to [input]")
```
Comment (Jordie0608) [inline code/modules/admin/verbs/randomverbs.dm]: Also needs a log_admin()

**Comment by WJohn** [comment]
:+1: x :100: 
Been wanting this for a while.

**Comment by Dorsisdwarf** [comment]
:+1:

**Pull request merged**